dapps/node-health/Cargo.toml

```diff
@@ -9,7 +9,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 futures = "0.1"
 futures-cpupool = "0.1"
 log = "0.3"
-ntp = "0.2.0"
+ntp = "0.3.0"
```

Comment:
Should this be 0.3.1?

Reply:
last 0 doesn't matter. we also use a lockfile